### ISSUE TITLE
Feature Request: Add configuration option to modify port-forwarding 

### ISSUE THREAD

**Issue by UnsafePointer** [veertuinc/packer-builder-veertu-anka#20]
We would like to enable port-forwarding with this Packer plugin as explained in https://ankadocs.veertu.com/docs/anka-cli/creating-templates-and-tags/#vm-networking

**Comment by NorseGaud** [comment]
Thanks for the feature request @UnsafePointer 
We're working diligently to have Anka CLI command/option parity throughout all of our plugins. You can expect to have this as a feature of the packer builder in the near future.

**Comment by marco-m-pix4d** [comment]
@UnsafePointer 

Our current workaround is to use a shell-local provisioner in the Packer configuration file:

```
{
      "type": "shell-local",
      "command": "anka modify '{{ user `vm-name` }}' add port-forwarding --host-port 0 --guest-port 22 ssh"
}
```

**Comment by NorseGaud** [comment]
Will be included in v1.6.0
